Translations/devilutionx.pot

```diff
@@ -14,311 +14,392 @@ msgstr ""
 "Last-Translator: \n"
 "Language-Team: \n"
 "X-Poedit-SearchPath-0: Source\n"
+"X-Poedit-Flags-xgettext: -c --no-location\n"
```

Comment:
We should consider if we want to pick up all comments (`-c`). Or use the default comment tag: `/* TRANSLATORS: text */`